static/js/domain-registration.js

```diff
@@ -76,17 +76,26 @@
                 modal.classList.add("is-visible");
 
                 const requestedDomain = document.querySelector(".js-subdomain-value");
-                const requestedDomainPreview = document.querySelectorAll(".js-modal-domain-registration-confirmation-domain-preview");
+                const requestedDomainPreview = document.querySelector(".js-modal-domain-registration-confirmation-domain-preview");
                 const requestedDomainPreviewEnding = document.querySelector(".js-modal-domain-registration-confirmation-domain-ending")
                 
                 // Preview the domain the user requested
-                requestedDomainPreview.forEach( preview => {
-                    preview.textContent = requestedDomain.value;
-                })
```

Comment:
Revert this change to restore that funcitonality (as called out in the [other comment](https://github.com/mozilla/fx-private-relay/pull/1109/files#r714451468)).

Reply:
Haaa that was silly; I was looking for element accessors (`[0]`, `[1]`, etc.) and just mentally ignored this `forEach` :joy:  Fixed in c5dfbc1 (which I've rebased already to avoid accidental merges; hope that doesn't interfere with local checkouts.)